lib/checkPullRequestLabels.js

```diff
@@ -16,286 +16,20 @@
  * @fileoverview File to handle checks when a PR gets labeled.
  */
 const {
-  LABELS_EXCLUDED_FROM_CODEOWNER_ASSIGNMENT,
   DATASTORE_LABEL,
   HOTFIX_LABEL,
   OLD_BUILD_LABEL,
-  getProjectOwnerFromLabel,
-  getChangelogLabelFromPullRequest,
-  isChangelogLabel,
   pingAndAssignUsers,
-  isUserCollaborator,
   checkPrIsStale
 } = require('./utils');
 const {
   releaseCoordinators,
   oppiaReleaseCoordinators,
-  oppiaDevWorkflowTeam,
   serverJobAdmins,
-  teamLeads
 } = require('../userWhitelist.json');
-const DEFAULT_CHANGELOG_LABEL = 'REVIEWERS: Please add changelog label';
-const PR_LABELS = ['dependencies', 'stale', DEFAULT_CHANGELOG_LABEL];
+const PR_LABELS = ['dependencies', 'stale'];
 const utilityModule = require('./utils');
-/**
- * This function tests a changelog label against a regex.
- *
- * @param {String} changelogLabel
- */
-const matchChangelogLabelWithRegex = function (changelogLabel) {
-  // Please refer to https://regex101.com/r/Eo976S/3 for explanation
-  // of the below regular expression.
-  const regExpForChangelogLabel = (
-    /^([pP][rR] [cC][hH][aA][nN][gG][eE][lL][oO][gG]: .* -- @\s*\w+\s*)/);
-  return regExpForChangelogLabel.test(changelogLabel);
-};
-
-/**
- * This function checks if the project owner can be assigned to a PR.
- *
- * @param {import('probot').Octokit.PullsGetResponse} pullRequest
- * @param {String} changelogLabel
- */
-const canAssignProjectOwner = (pullRequest, changelogLabel) => {
-  // Check if the changelog label is excluded from codeowner assignment.
-  const isLabelExcluded = LABELS_EXCLUDED_FROM_CODEOWNER_ASSIGNMENT.some(
-    (label) => changelogLabel.startsWith(label)
-  );
-  if (isLabelExcluded) {
-    return true;
-  }
-  const projectOwner = getProjectOwnerFromLabel(changelogLabel);
-  const reviewers = pullRequest.requested_reviewers.map(
-    (reviewer) => reviewer.login
-  );
-  const canReview = reviewers.includes(projectOwner);
-
-  return canReview;
-};
-
-/**
- * This function assigns pull request to the appropriate reviewers.
- *
- * @param {import('probot').Context} context
- */
-const assignAllCodeowners = async (context) => {
```

Comment:
Don't we still want this functionality, in general?

Reply:
Yep, moved back